pkg/sysregistriesv2/system_registries_v2_test.go

```diff
@@ -83,43 +83,84 @@ func TestMirrors(t *testing.T) {
 	assert.True(t, reg.Mirrors[1].Insecure)
 }
 
-func TestRefMatchesPrefix(t *testing.T) {
+func TestRefMatchingSubdomainPrefix(t *testing.T) {
 	for _, c := range []struct {
 		ref, prefix string
-		expected    bool
+		expected    int
+	}{
+		// Check for subdomain matches
+		{"docker.io", "*.io", -1},
```

Comment:
This should match.

Reply:
will do. I was returning anything not of the format `foo.com/bar` as match fail. I'll change this.